iml-docker.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=iml-docker
+Requires=docker.service
+After=docker.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+StandardError=null
```

Comment:
We don't want to send output to journal?

Reply:
I'm not sure yet. I need to play around with a bit to figure out the best thing to do.

Reply:
Changed to journal output.